src/js/controller/settings/exportimage/GifExportController.js

```diff
@@ -177,7 +177,7 @@
         link : imageUrl,
         shortLink : this.shorten_(imageUrl, URL_MAX_LENGTH, '...')
       });
-      this.uploadStatusContainerEl.innerHTML = `${this.i18n.gifExportSettingSectionYourImageAvailableAt()} ${linkHtml}`;
+      this.uploadStatusContainerEl.innerHTML = this.i18n.gifExportSettingSectionYourImageAvailableAt() + ' :' + linkHtml;
```

Comment:
Why stop using the backticks? Was this a merge mistake?
```suggestion
      this.uploadStatusContainerEl.innerHTML = `${this.i18n.gifExportSettingSectionYourImageAvailableAt()} :${linkHtml}`;
```